.github/workflows/nightly-cron.yml

```diff
@@ -0,0 +1,18 @@
+# Nightly Cron workflow for GitHub Actions
+name: "⏰ Nightly Cron"
+on:
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
When does the cron job run? Do we want to run it at midnight eastern time since our initial, core target group / audience lives on the East coast? If so, how should we handle the switch from eastern standard to daylight time?

Reply:
GitHub uses UTC since users from all over the world can use their services, and UTC is the average of all the timezones in the world.